docs/sources/mimir/references/glossary.md

```diff
@@ -21,23 +25,23 @@ A chunk is an object containing encoded timestamp-value pairs for one series.
 
 Churn is the frequency at which series become idle.
 
-A series becomes idle once it's no longer exported by the monitored targets.
-Typically, series become idle when a monitored target process or node gets terminated.
+A series becomes idle after it’s no longer exported by the monitored targets.
+Typically, series become idle after a monitored target process or node gets terminated.
```

Comment:
What's the benefit of changing `it's` to it’s`? I see no visual difference in the generated HTML.

Reply:
Habit, really. I use `’` for language and `'` for code or frontmatter.